doc/release-notes/upgrading.en.rst

```diff
@@ -64,6 +64,8 @@ Configuration changes
 ---------------------
 The following incompatible changes to the configurations have been made in this version of ATS.
 
+The following changes have been made to the :file:`records.yaml` file:
```

Comment:
Sounds a bit odd because records.yaml is technically a new config file, mentioning file names is a good idea though. Let's adjust it (if needed) on another PR that explains the transition.

Reply:
What do you think now? Now, via another PR, the change to `records.yaml` is mentioned before this.

Reply:
Now it looks fine.